themes/default/content/docs/get-started/install/_index.md

```diff
@@ -43,6 +45,15 @@ Subsequent updates can be installed in the usual way:
 $ brew upgrade pulumi
 ```
 
+### Community Homebrew
+
+A Pulumi formula is available on the Community Homebrew. If you do not have the Pulumi tap installed, then you can
```

Comment:
Link to the community formula on github?

Reply:
That'll go in the accompanying blog post - I don't want a CTA that takes people away from the install :)